runtime/test/src/lib.rs

```diff
@@ -483,6 +483,7 @@ impl pallet_parachain_staking::Config for Runtime {
     type CollatorSessionRegistration = Session;
     type CollatorPayoutDustHandler = TokenManager;
     type WeightInfo = pallet_parachain_staking::weights::SubstrateWeight<Runtime>;
+    type MaxCandidates = ConstU32<100>;
```

Comment:
I don't understand the value of this limit. How many candidates can we have? Why is 100 safe?